File: modules/04_privacy_preserving_ai/code_samples/android_derived_data/StressHeuristic.kt
enum class StressLevel { LOW, MEDIUM, HIGH }

fun inferStress(features: HeartRateFeatures): StressLevel =
    when {
        features.stdDev > 15 && features.mean > 80 -> StressLevel.HIGH
        features.stdDev > 8 -> StressLevel.MEDIUM
        else -> StressLevel.LOW
    }

File: modules/04_privacy_preserving_ai/code_samples/ios_derived_data/StressHeuristic.swift
// A simple heuristic “model” using features:
enum StressLevel: String {
    case low, medium, high
}

func inferStress(from features: HeartRateFeatures) -> StressLevel {
    if features.stdDev > 15 && features.mean > 80 {
        return .high
    } else if features.stdDev > 8 {
        return .medium
    } else {
        return .low
    }
}
